can-ajax-test.js

```diff
@@ -50,6 +50,56 @@ if (hasLocalServer) {
 		});
 	});
 
+	QUnit.test("Body payload should append to the url query string when PATCH method is used (#75)", function (assert) {
```

Comment:
is this mistitled?  The issue you're fixing says the opposite of this.

Reply:
I have refactored the test name to better reflect the PR